default.hbs

```diff
@@ -106,6 +106,18 @@
             if (stripe == 'success') {
                 $('body').addClass("checkout-success");
             }
+            if (stripe == 'billing-update-success') {
+                window.location = '{{@site.url}}/account/?stripe=billing-update-success-account';
+            }
+            if (stripe == 'billing-update-cancel') {
+                window.location = '{{@site.url}}/account/?stripe=billing-update-cancel-account';
+            }
+            if (stripe == 'billing-update-success-account') {
```

Comment:
@peterzimon had to do double redirects here because the handlers are global... so we have to have separate parameter names for when we catch the on front page and on account page.